src/sgame/sg_bot_util.cpp

```diff
@@ -309,10 +309,9 @@ float BotGetBaseRushScore( gentity_t *ent )
 	// return average value on which other parameters can weight
 	float rush_score = 0.5;
 
-	float self_value = static_cast<float>( ent->client->ps.persistant[PERS_CREDIT]
-		+ BotValueOfWeapons( ent )
-		+ BotValueOfUpgrades( ent )
-		+ BG_Class( ent->client->ps.stats[ STAT_CLASS ] )->price );
+	float self_value = static_cast<float>(
+		ent->client->ps.persistant[PERS_CREDIT]
+			+ BG_GetPlayerPrice( ent->client->ps ) );
```

Comment:
Does not work. the class is missing.

Reply:
I don't think it is

Reply:
right, it's done with that function. Still not convinced, though, since current code is team agnostic, and the proposed patch is not.

Reply:
LGTM anyway

Reply:
LGTM